libs/bluegl/src/BlueGLLinuxOSMesa.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <GL/gl.h>
+#include <dlfcn.h>
+#include <string.h>
+
+namespace bluegl {
+
+namespace {
+using ProcAddressFunc = void*(*)(char const* funcName);
+}
+
+struct Driver {
+    ProcAddressFunc OSMesaGetProcAddress;
+    void* library;
+} g_driver = {nullptr, nullptr};
+
+bool initBinder() {
```

Comment:
Where do these functions get used?

Reply:
these are the GL api functions. See the parallel for regular linux - [BlueGLLinux][link].

[link]: https://github.com/google/filament/blob/main/libs/bluegl/src/BlueGLLinux.cpp